api_tabular/utils.py

```diff
@@ -90,70 +106,125 @@ def swagger_parameters(resource_columns):
             'schema': {
                 'type': 'string'
             }
-        }
+        },
     ]
+    # expected python types are: string, float, int, bool, date, datetime, json
+    # see metier_to_python here: https://github.com/datagouv/csv-detective/blob/master/csv_detective/explore_csv.py
+    # see cast for db here: https://github.com/datagouv/hydra/blob/main/udata_hydra/analysis/csv.py
     for key, value in resource_columns.items():
-        parameters_list.extend(
-            [
-                {
-                    'name': f'sort ascending {key}',
-                    'in': 'query',
-                    'description': f'{key}__sort=asc.',
-                    'required': False,
-                    'schema': {
-                        'type': 'string'
-                    }
-                },
-                {
-                    'name': f'sort descending {key}',
-                    'in': 'query',
-                    'description': f'{key}__sort=desc.',
-                    'required': False,
-                    'schema': {
-                        'type': 'string'
-                    }
-                }
-            ]
-        )
-        if value['python_type'] == 'string':
+        if "exact" in TYPE_POSSIBILITIES[value['python_type']]:
             parameters_list.extend(
                 [
                     {
-                        'name': f'exact {key}',
+                        'name': f'{key}__exact=value',
                         'in': 'query',
-                        'description': f'{key}__exact=value.',
+                        'description': f'Exact match in column: {key}',
                         'required': False,
                         'schema': {
                             'type': 'string'
                         }
                     },
+                ]
+            )
+        if "differs" in TYPE_POSSIBILITIES[value['python_type']]:
+            parameters_list.extend(
+                [
                     {
-                        'name': f'contains {key}',
+                        'name': f'{key}__differs=value',
                         'in': 'query',
-                        'description': f'{key}__contains=value.',
+                        'description': f'Differs from in column: {key}',
                         'required': False,
                         'schema': {
                             'type': 'string'
                         }
-                    }
+                    },
+                ]
+            )
+        if "in" in TYPE_POSSIBILITIES[value['python_type']]:
+            parameters_list.extend(
+                [
+                    {
+                        'name': f'{key}__in=(value1,value2,...)',
```

Comment:
I wasn't expecting the parenthesis in the list. But I don't know if has some utility or not so we can keep it if you prefer

Reply:
It could also be `in=value1,value2,...`, I kept it for readability reasons but I don't know whether  that is standard

Reply:
@geoffreyaldebert any opinion on this?

Reply:
without parenthesis i think it is more common